Makefile

```diff
@@ -25,7 +25,7 @@ unannotated_pyright:
 	python scripts/run-pyright.py --unannotated
 
 ruff:
-	-ruff --fix .
+	-ruff check --fix .
```

Comment:
A breadcrumb for other reviewers: `ruff <path>` was deprecated in favor of `ruff check <path>`. See https://github.com/astral-sh/ruff/releases/tag/v0.3.0.